src/main/java/com/liferay/faces/util/el/internal/ProductMap.java

```diff
@@ -30,7 +31,14 @@
 public class ProductMap implements Map<Product.Name, Product>, Serializable {
 
 	// serialVersionUID
-	private static final long serialVersionUID = 8051373438471283130L;
+	private static final long serialVersionUID = 2730600575388304439L;
+
+	// Private Final Data Members
+	private final ProductFactory productFactory;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.